### PR TITLE
Deploy SQLFlow client for all OSes

### PR DESCRIPTION
**Pull request by wangkuiyi**
After the merge of https://github.com/sql-machine-learning/sqlflow/pull/2236, could we merge this PR.

This PR enhanced the release of SQLFlow client for macOS, Linux, and Widows.

### CODE REVIEW

File: scripts/travis/deploy_client.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+echo "Build cmd/sqlflow into /tmp ..."
+cd $TRAVIS_BUILD_DIR
+go generate ./...
+GOBIN=/tmp go install ./cmd/sqlflow
+
+echo "Install AWS client for $TRAVIS_OS_NAME ..."
+if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    curl -s "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"
+    sudo installer -pkg AWSCLIV2.pkg -target /
+elif [[ "$TRAVIS_OS_NAME" == "macos" ]]; then
```
Comment (lhw362950217) [inline scripts/travis/deploy_client.sh]: macos -> linux ?

Comment (wangkuiyi) [reply scripts/travis/deploy_client.sh]: Corrected.  I am curious why the CI passed with this error.

Comment (lhw362950217) [reply scripts/travis/deploy_client.sh]: It skipped the deployment because we are in a PR-build, according to the log.

**Review by lhw362950217 — APPROVED** [review]
LGTM

**Pull request merged**